corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -151,10 +151,19 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
             </div>
           {% endif %}
           <div id="submit-button" class="form-actions form-group noprint-sub-container" data-bind="visible: showSubmitButton">
-            <div data-bind="css: submitClass">
-              <button class="submit btn btn-lg btn-primary"
+            <div data-bind="
+              css: submitClass,
+              style: {
+                'bottom':isAnchoredSubmitStyle && {{ request.couch_user.can_edit_data|yesno:'true,false' }} ? '30px' : '' {# data preview bar #}
```

Comment:
Yes it does. `request.couch_user.can_edit_data` returns `True` with a capital "T". The pipe interprets that and returns `true` with a lower case "t" which knockout can process. Otherwise it errors.